src/classes/Timesheet.ts

```diff
@@ -91,6 +90,9 @@ export class Timesheet {
 
   public numberOfRemainingWorkDays = (): number => {
     let workingDaysLeft: number = this.weekDaysBetweenDates(this.todaysDate, this.timesheetEndDate);
+    if (workingDaysLeft === 0) {
+      return 0;
```

Comment:
ℹ️ This missing code is what fixed the bug. If there are already 0 days left, stop and return 0 -- don't do the additional check.